File: 1-easy/problems/2923-find-champion-i/solution.ts
function findChampion(grid: any): boolean | number | string | any {
  """
  :type grid: List[List[int]]
  :rtype: int
  """
  return next(u for u in Array.from({length: (grid}, (_, i) => i).length) if sum(grid[u]) == (grid).length-1);
}

export { findChampion };

File: 1-easy/problems/2923-find-champion-i/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def findChampion(self, grid):
        """
        :type grid: List[List[int]]
        :rtype: int
        """
        return next(u for u in range(len(grid)) if sum(grid[u]) == len(grid)-1)
